spec/support/controller_spec_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+shared_examples_for 'skip_sentry_404' do
+  before do
+    allow_any_instance_of(described_class).to receive(:authenticate) do
+      raise Common::Exceptions::RecordNotFound, 'some_id'
+    end
+
+    controller_klass = described_class
+    Rails.application.routes.draw do
+      get '/authenticate' => "#{controller_klass.to_s.underscore.gsub('_controller', '')}#authenticate"
```

Comment:
`/authenticate` and the `#authenticate` action is just a fake route right?  I got stuck for a moment thinking you were mocking out the existing `ApplicationController#authenticate` action.

I'd recommend renaming it to be more obvious, like `/fake_route` or something.

Reply:
i am mocking the `ApplicationController#authenticate` action otherwise things like `EVSSClaimsBaseController#authorize_user` will fail, i can change the route path though